agent/stats/utils_windows_test.go

```diff
@@ -28,41 +27,27 @@ import (
 
 func TestDockerStatsToContainerStatsZeroCoresGeneratesError(t *testing.T) {
 	numCores = uint64(0)
-	jsonStat := fmt.Sprintf(`
-		{
-			"cpu_stats":{
-				"cpu_usage":{
-					"total_usage":%d
-				}
-			}
-		}`, 100)
+	inputJsonFile, _ := filepath.Abs("./windows_test_stats.json")
```

Comment:
the reason why I didn't change this was - it would be costly to open/read a file on disk for each test. Do you think it's possible to combine below test with this one?

Reply:
+1, windows anything is already slow

Reply:
I combined all tests requiring stats.json into a single test so we only read the file once.